and-cli-deploy-jenkins.js

```diff
@@ -106,7 +106,7 @@ require("./command-runner").run(async () => {
             const configPath = jenkins.getConfigPath();
             const userPrompt = prompt.getPrompt();
 
-            await userPrompt.confirmOrExit(
+            await prompt.confirmOrExit(
```

Comment:
😛 Sorry this bit ya. The refactored prompt in the typescript branch cleans this up you consumers have a single object to call for a standard question prompt _or_ the `confirmOrExit` functionality, and it manages the prompt instance internally.